ouroboros-network/src/Ouroboros/Network/Diffusion/Policies.hs

```diff
@@ -51,6 +51,14 @@ peerMetricsConfiguration = PeerMetricsConfiguration {
     maxEntriesToTrack = 180
   }
 
+
+-- | Time required to pass in order for an inbound peer to be available for the
+-- outbound governor.  The default values is 15mins.
+--
+inboundPeerDelay :: DiffTime
+inboundPeerDelay = -15 * 60
```

Comment:
Oh, but it seems I forgot to multiply it by `(-1)` in the right place:
https://github.com/intersectmbo/ouroboros-network/blob/a31646da8edd33ec36d36cdf69ce3066a8fe6aa7/ouroboros-network-framework/src/Ouroboros/Network/InboundGovernor.hs#L183

Reply:
We ought to do this:
```hs
                 case OrdPSQ.atMostView
                       ((-inboundMaturePeerDelay) `addTime` time)
                       (igsFreshDuplexPeers state) of
                   ([], _)   -> retry
                   (as, pq') -> let m = Map.fromList ((\(addr, _p, v) -> (addr, v)) <$> as)
                                in pure $ MaturedDuplexPeers m pq'
```
where
```
inboundMaturePeerDelay :: DiffTime
inboundMaturePeerDelay = 15 * 60
```

Reply:
Fixed.

Reply:
I found one more problem, which is fixed in the commit:


> inbound-governor: wakeup inbound governor on regular intervals
> 
> To push peers that become mature, we need to wakeup the inbound governor
> on regular intervals.  Long inactivity periods could happen on nodes
> which start at the edge of network, when they have very few inbound
> connections.  The inbound governor is woken up every 30s.

I also had to modify the peer-sharing unit test so that it passed.